packages/@o3r/dynamic-content/src/services/dynamic-content/dynamic-content.service.ts

```diff
@@ -46,7 +44,7 @@ export class DynamicContentService {
     if (this.cmsOnlyAssetsPath && assetPath) {
       return assetPath.startsWith('/') ? assetPath : `${this.cmsOnlyAssetsPath.replace(/\/$/, '')}/${assetPath}`;
     }
-    return this.getContentPath(this.mediaFolder ? `${this.mediaFolder}/${this.normalizePath(assetPath)}` : assetPath);
+    return this.getContentPath(`${this.mediaFolderName}/${this.normalizePath(assetPath)}`);
```

Comment:
```suggestion
    return this.getContentPath(`${
      this.mediaFolderName ? this.mediaFolderName + '/' : ''
    }${this.normalizePath(assetPath)}`);
```
prefixing by `/` could maybe cause trouble if the normalized path starts with `..`

Reply:
The normalize path called by the getContentPath prevents that
![image](https://github.com/AmadeusITGroup/otter/assets/122815580/44c8be24-a903-470d-95ad-bf1d42b9aea7)